From c22e1c5003e78917a138b48beff10d39b2c3c7bb Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Mon, 30 Nov 2015 10:19:35 +0800
Subject: [PATCH] MLK-11917-1 media: output: pxp_v4l2: remove some duplicate
 dead code

The code slice in the 'v4l2_fmt_to_pxp_fmt()' function:
"
        else if (v4l2_pix_fmt == V4L2_PIX_FMT_RGB555)
                pxp_fmt = PXP_PIX_FMT_RGB555;
"
appears twice continuously. And the second code slice
will become dead code which is never executed. So
remove the second one.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
(cherry picked from commit 94c7651e9f0b30e137571362e9bfb662ccabf53b)
---
 drivers/media/platform/mxc/output/mxc_pxp_v4l2.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/media/platform/mxc/output/mxc_pxp_v4l2.c b/drivers/media/platform/mxc/output/mxc_pxp_v4l2.c
index f070e62..3252f7b 100644
--- a/drivers/media/platform/mxc/output/mxc_pxp_v4l2.c
+++ b/drivers/media/platform/mxc/output/mxc_pxp_v4l2.c
@@ -117,8 +117,6 @@ static unsigned int v4l2_fmt_to_pxp_fmt(u32 v4l2_pix_fmt)
 		pxp_fmt = PXP_PIX_FMT_RGB565;
 	else if (v4l2_pix_fmt == V4L2_PIX_FMT_RGB555)
 		pxp_fmt = PXP_PIX_FMT_RGB555;
-	else if (v4l2_pix_fmt == V4L2_PIX_FMT_RGB555)
-		pxp_fmt = PXP_PIX_FMT_RGB555;
 	else if (v4l2_pix_fmt == V4L2_PIX_FMT_YUV420)
 		pxp_fmt = PXP_PIX_FMT_YUV420P;
 	else if (v4l2_pix_fmt == V4L2_PIX_FMT_YUV422P)
-- 
1.8.0

